xlml/utils/metric.py

```diff
@@ -598,6 +598,7 @@ def process_metrics(
     task_metric_config: metric_config.MetricConfig,
     task_gcp_config: gcp_config.GCPConfig,
     use_startup_script: bool = False,
+    file_location: str = None,
```

Comment:
Wondering if this is the right change to support this feature. We could have lots of environment variables in the future, and we cannot just adding more if/else here.

Reply:
Sad thing is that `task_metric_config` currently is decided at DAG creation time, it's hard to update the `task_metric_config.profile.file_locations`.  Let me do a try to see if I can overwrite the `task_metric_config` s.t. we can get rid of passing `file_location` here.